src/settings.ts

```diff
@@ -305,6 +330,41 @@ export const BURN_VENUE_TEMPLATES: Array<Template> = [
   },
 ];
 
+// @debt Refactor this constant into types/templates or similar?
+// NOTE: The Map object holds key-value pairs and remembers the original insertion order of the keys
+// Same can be accomplished by a regular object if created through Object.fromEntries()
+export const BURN_VENUE_TEMPLATES_MAP: Map<VenueTemplate, Template> = new Map(
```

Comment:
The only benefit to use Map here would be to pass object as key which is not needed, the key is string and this can be simplify to plain object instead.

Reply:
`plain object` doesn't guarantee order of properties unless is created by `Object.fromEntries()` as per the comment, which at the end is as not much simpler than using `Map`.

Considering Maps guarantee ordering of entries, I disagree the only benefit would be to pass some argument to the Map.

Reply:
There is a work in progress of specifying order of properties in objects (more specifically, how they're being manipulated by different language elements) https://github.com/tc39/proposal-for-in-order, but for now, using `Map` shows a clear signal of the intent 
> The Map object holds key-value pairs and remembers the original insertion order of the keys. 
--
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map

Reply:
Fair enough, just saw the comment above the const.